src/USER-MISC/fix_bond_react.cpp

```diff
@@ -207,7 +208,9 @@ FixBondReact::FixBondReact(LAMMPS *lmp, int narg, char **arg) :
 
     iarg++;
 
-    rxn_name[rxn] = arg[iarg++];
+    int n = strlen(arg[iarg]) + 1;
+    if (n > MAXLINE) error->all(FLERR,"Reaction name (react-ID) is too long (limit: 256 characters)");
```

Comment:
if checking against the maximum storage capacity with `strlen()`, there is not need to use `strncpy()` as the call becomes the equivalent of `strcpy()` with how this is currently coded. However, in the header the rxn_name size is given as an explicit number and not `MAXLINE` like here. Either both should be using the same explicit number, or you could create a symbolic constant inside the header, eg. using an enum.

Reply:
thanks, this has been corrected. (unless you have a strong preference for using an enum, rather than a #define in the header)